source/d3d11/d3d11_device_context.hpp

```diff
@@ -183,22 +183,32 @@ struct D3D11DeviceContext : ID3D11DeviceContext3
 	virtual void STDMETHODCALLTYPE SetMarkerInt(LPCWSTR pLabel, INT Data) override;
 	virtual void STDMETHODCALLTYPE BeginEventInt(LPCWSTR pLabel, INT Data) override;
 	virtual void STDMETHODCALLTYPE EndEvent() override;
-	#pragma endregion
-	#pragma region ID3D11DeviceContext3
+#pragma endregion
+#pragma region ID3D11DeviceContext3
 	virtual void STDMETHODCALLTYPE Flush1(D3D11_CONTEXT_TYPE ContextType, HANDLE hEvent) override;
 	virtual void STDMETHODCALLTYPE SetHardwareProtectionState(BOOL HwProtectionEnable) override;
 	virtual void STDMETHODCALLTYPE GetHardwareProtectionState(BOOL *pHwProtectionEnable) override;
-	#pragma endregion
+#pragma endregion
 
 	void log_drawcall(UINT vertices);
+	ID3D11DepthStencilView *copy_depthstencil(ID3D11DepthStencilView *depthstencil);
+	com_ptr<ID3D11Texture2D> copy_depth_texture(D3D11_TEXTURE2D_DESC texture_desc, ID3D11Texture2D *depthtexture);
+	bool check_depthstencil(ID3D11DepthStencilView* pDepthStencilView, com_ptr<ID3D11Texture2D> texture, D3D11_TEXTURE2D_DESC texture_desc);
 	void set_active_depthstencil(ID3D11DepthStencilView* pDepthStencilView);
+	void set_active_OM_depthstencil(ID3D11DepthStencilView* pDepthStencilView);
+	void set_active_cleared_depthstencil(ID3D11DepthStencilView* pDepthStencilView);
 
 	void clear_drawcall_stats();
 
 	LONG _ref = 1;
+	unsigned int _vertices_treshold = 10000;
```

Comment:
I don't see a reason behind this field. It looks to be a constant, so may as well put it in as a constant where it's used in the cpp file.